src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -409,12 +409,14 @@ private async Task<PackageSpec> GetPackageSpecAsync(ISettings settings)
             string enableAudit = _vsProjectAdapter.BuildProperties.GetPropertyValue(ProjectBuildProperties.NuGetAudit);
```

Comment:
What would our docs say? Use `NuGetAudit` or use `NuGetAuditRequired`? 

I'm not a huge fan of having the extra property to just enable audit. 
Since we're not really shipping audit by default very soon, IMO, it's ok if NuGetAudit is the only configuration knob.

Do we expect that people will ever need to "enable" NuGet Audit themselves? Since it's enabled by default.

Reply:
> I'm not a huge fan of having the extra property to just enable audit.

There are customers who use alternative vulnerability scanners, so will probably want to disable NuGet's built-in scanning. For example within Microsoft, we already have Component Governance, who has more vulnerability sources than just GitHub's Advisory Database.

So, I see `<NuGetAudit>false` as a better alternative than `<NoWarn>NU1901;NU1902;NU1903;NU1904;$(NuWarn)`.

> Since we're not really shipping audit by default very soon

yes we are, in both .NET 8 SDK and VS17.8 it's on by default.

> Do we expect that people will ever need to "enable" NuGet Audit themselves? Since it's enabled by default.

Ok, now I don't understand your previous "we're not really shipping audit by default very soon" statement

> What would our docs say? Use NuGetAudit or use NuGetAuditRequired?

If you don't want to use NuGet to scan restored packages for known vulnerabilities, set the `NuGetAudit` property to `false`.

If you want a warning that NuGet Audit was unable to run, due to no package sources providing a list of packages with known vulnerabilities, then set the `NuGetAuditRequired` property to `true`. If you want restore to fail, also add `NU1905` to `WarningsAsErrors`.

Reply:
> yes we are, in both .NET 8 SDK and VS17.8 it's on by default.

That was a typo on my end. 

> If you want a warning that NuGet Audit was unable to run, due to no package sources providing a list of packages with known vulnerabilities, then set the NuGetAuditRequired property to true. If you want restore to fail, also add NU1905 to WarningsAsErrors.

How many people would want to do that? Have the failure? I understand the "miss" in the scenario, wanting audit explicitly, but not getting audit cause sources were not configured, but is that scenario important enough that we want to introduce what I feel like an imperfect solution in V1?

Reply:
FWIW I think the extra complexity of having two properties isn't worth it.